### PR TITLE
WIP: Chromium bump

### PR DESCRIPTION
**Pull request by ryantrinkle**
###### Motivation for this change
Chromium doesn't seem to build without https://github.com/NixOS/nixpkgs/commit/b746b4e1efc2c6662667a9e68265c42cf10a81a9

I could use some help figuring out how to properly test this.

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/browsers/chromium/common.nix [hunk 1]
```diff
@@ -130,7 +130,7 @@ let
       }' gpu/config/gpu_control_list.cc
 
       patchShebangs .
-    '' + optionalString (versionAtLeast version "52.0.0.0") ''
+    '' + optionalString (versionAtLeast version "52.0.0.0" && !(versionAtLeast version "58.0.0.0")) ''
```
Comment (Mic92) [inline pkgs/applications/networking/browsers/chromium/common.nix]: optionalString (versionAtLeast version "52.0.0.0" && versionOlder version "58.0.0.0")

Comment (ryantrinkle) [inline pkgs/applications/networking/browsers/chromium/common.nix]: Looks good, thanks!

**Comment by ryantrinkle** [comment]
Looks like this still fails to build, now with:

```
[505/26989] ACTION //chrome/browser/resources/md_downloads:vulcanize(//build/toolchain/linux:x64)
FAILED: gen/chrome/browser/resources/md_downloads/vulcanized.unbuilt.html gen/chrome/browser/resources/md_downloads/crisper.js 
python ../../chrome/browser/resources/vulcanize_gn.py --host downloads --html_in_file downloads.html --html_out_file vulcanized.unbuilt.html --js_out_file crisper.js --input ../../chrome/browser/resources/md_downloads --out_folder gen/chrome/browser/resources/md_downloads --depfile gen/chrome/browser/resources/md_downloads/vulcanized.unbuilt.html.d --insert_in_head \<base\ href=chrome://downloads\>
../../third_party/node/linux/node-linux-x64/bin/node ../../third_party/node/node_modules/vulcanize/bin/vulcanize --exclude chrome://resources/html/polymer.html --exclude web-animations-next-lite.min.js --exclude load_time_data.js --exclude strings.js --exclude text_defaults.css --exclude text_defaults_md.css --inline-css --inline-scripts --strip-comments --redirect "chrome://resources/cr_elements/|../../ui/webui/resources/cr_elements" --redirect "chrome://resources/css/|../../ui/webui/resources/css" --redirect "chrome://resources/html/|../../ui/webui/resources/html" --redirect "chrome://resources/js/|../../ui/webui/resources/js" --redirect "chrome://resources/polymer/v1_0/|../../third_party/polymer/v1_0/components-chromium" --out-request-list /tmp/nix-build-chromium-59.0.3071.71.drv-0/chromium-59.0.3071.71/out/Release/gen/chrome/browser/resources/md_downloads/vulcanized.unbuilt.html_requestlist.txt --redirect "/|/tmp/nix-build-chromium-59.0.3071.71.drv-0/chromium-59.0.3071.71/chrome/browser/resources/md_downloads" --redirect "chrome://downloads/|/tmp/nix-build-chromium-59.0.3071.71.drv-0/chromium-59.0.3071.71/chrome/browser/resources/md_downloads" /downloads.html failed: /bin/sh: ../../third_party/node/linux/node-linux-x64/bin/node: No such file or directory

Traceback (most recent call last):
  File "../../chrome/browser/resources/vulcanize_gn.py", line 207, in <module>
    main(sys.argv[1:])
  File "../../chrome/browser/resources/vulcanize_gn.py", line 202, in main
    _vulcanize(args.input, args)
  File "../../chrome/browser/resources/vulcanize_gn.py", line 150, in _vulcanize
    args.html_in_file)])
  File "../../third_party/node/node.py", line 28, in RunNode
    raise
TypeError: exceptions must be old-style classes or derived from BaseException, not NoneType
ninja: build stopped: subcommand failed.
```

**Comment by joachifm** [comment]
See also https://github.com/NixOS/nixpkgs/pull/25940, which claims to fix the build.

**Comment by ryantrinkle** [comment]
@joachifm Ah, ok, cool; that definitely looks like it fixes the new error I was getting.  Perhaps I picked the wrong version as the cutoff for the utils.h stuff, or perhaps it's unnecessary for some other reason.  I'll look into this more if this issue is run into again.

**Pull request closed without merging**